pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSConfigurator.java

```diff
@@ -66,4 +68,8 @@ public ResourceManager getDefaultConfig() {
                         UNKNOWN_MODEL_PATH, UNKNOWN_LEX_PATH, BASELINE_LEX_PATH, MIKHEEV_LEX_PATH};
         return new ResourceManager(generateProperties(props));
     }
+
```

Comment:
Why is this method needed (i.e. why not just inline "new ResourceManager(configFile)")?